### PR TITLE
Cleanup documentation and modifiers

### PR DESCRIPTION
**Pull request by holgerfriedrich**
- ~~Cleanup Javadoc annotations (removing brackets from {@return statements})~~
- Change modifiers according to JLS (~~mainly removing final from static methods, as static methods are implicitly final, but also~~ order of modifiers)
- Minor other improvements

### CODE REVIEW

File: src/io/calimero/dptxlator/DPTXlator.java [hunk 1]
```diff
@@ -506,7 +506,7 @@ final String removeUnit(final String value)
 		return value.trim();
 	}
 
-	static final short ubyte(final int value)
+	static short ubyte(final int value)
```
Comment (bmalinowsky) [inline src/io/calimero/dptxlator/DPTXlator.java:509]: Putting final on static methods prevents hiding the method, its usage is fine.

File: src/io/calimero/dptxlator/DPTXlatorDateTime.java [hunk 2]
```diff
@@ -982,7 +982,7 @@ else if (++day == 1) {
 						int dow = DAYS.length - 1;
 						while (dow >= 0 && !DAYS[dow].toLowerCase().startsWith(prefix))
 							--dow;
-						final boolean anyday = dow == 0 && t.hasMoreTokens() && t.nextToken().equalsIgnoreCase("day");
+						final boolean anyday = dow == 0 && t.hasMoreTokens() && "day".equalsIgnoreCase(t.nextToken());
```
Comment (bmalinowsky) [inline src/io/calimero/dptxlator/DPTXlatorDateTime.java:985]: Switching the operands in cases like these seems unnecessary?

**Review by bmalinowsky — COMMENTED** [review]
Thanks, I added some comments.

File: src/io/calimero/dptxlator/DPTXlatorDateTime.java (hunk 2 above)
Comment (holgerfriedrich) [reply src/io/calimero/dptxlator/DPTXlatorDateTime.java:985]: Using ```"constant".eqals(somestringvariable)``` has the benefit that the string constant is never ```null``` and running ```equals()``` cannot cause a NPE. It cannot harm to write it in this order.

File: src/io/calimero/dptxlator/DPTXlator.java (hunk 1 above)
Comment (holgerfriedrich) [reply src/io/calimero/dptxlator/DPTXlator.java]: Rolled back as requested. AFAIR there are methods only declared static without the final keyword. Not sure if this is intended then.

File: src/io/calimero/GroupAddress.java [hunk 3]
```diff
@@ -207,7 +207,7 @@ public static void addressStyle(final Presentation style)
 	}
 
 	/**
-	 * {@return the address presentation style preset for all KNX group addresses}
```
[the remark this answers is not in the document]
Comment (holgerfriedrich) [reply src/io/calimero/GroupAddress.java]: The inline javadoc return statements were not properly recognized by my tooling (though technically fine) and I had not realized that you just changed it to be this way not too long ago.

**Comment by holgerfriedrich** [comment]
Hi @bmalinowsky, sorry for replying late.

I followed your advice and rolled back accordingly. Seems this PR is down to two minor changes....

File: src/io/calimero/dptxlator/DPTXlatorDateTime.java (hunk 2 above)
Comment (bmalinowsky) [reply src/io/calimero/dptxlator/DPTXlatorDateTime.java:985]: That's true; the thing is, if `nextToken` returns null something is really really wrong. So a reversal like this basically hardens against the langspec/compiler/interpreter.

**Comment by bmalinowsky** [comment]
Thanks for the PRs!

**Pull request merged**